build.gradle.kts

```diff
@@ -106,5 +106,7 @@ configurations.all {
     resolutionStrategy {
         force("ch.qos.logback:logback-core:1.4.14")
         force("ch.qos.logback:logback-classic:1.4.14")
+        force("org.springframework.cloud:spring-cloud-starter-openfeign:4.1.2")
```

Comment:
Is it no better to upgrade spring cloud to 2023.0.2 than force the version here?